karspexet/ticket/payment.py

```diff
@@ -104,3 +96,30 @@ def _send_mail_to_customer(self):
             [to_address],
             fail_silently=False,
         )
+
+class FakePaymentProcess(PaymentProcess):
+
+    def _charge_card(self):
+        if self.data['payment_success'] == 'true':
+            return True
```

Comment:
Seems this should just return void, aka `None` as it raises an exception instead of returning `False`.